lib/posix-netlink/Config.uk

```diff
@@ -0,0 +1,8 @@
+config LIBPOSIX_NETLINK
+	bool "posix-netlink: Abstraction for netlink protocols"
+	select LIBPOSIX_SOCKET
+	help
+	  This microlibrary allows other microlibraries to register a
```

Comment:
So, I've always thought that we use LK style for KConfig and I've even pointed it out to you in the past :sweat: . But recently it's been brought up to me that that is actually not the case and we actually do use tab instead of 2 whitespaces for the help message indentation.
CC: @michpappas